curriculum/challenges/english/14-responsive-web-design-22/learn-css-colors-by-building-a-set-of-colored-markers/61a489b8579e87364b2d2cdb.md

```diff
@@ -20,7 +20,7 @@ In the `.red` CSS rule, change the `background-color` property to `background`.
 Your `.red` CSS rule should have a `background` property with the value `rgb(255, 0, 0)`.
 
 ```js
-assert.include(new __helpers.CSSHelp(document).getStyle('.red').cssText, 'background: rgb(255, 0, 0)');
+assert.match(__helpers.removeWhiteSpace(code), /\.red\{.*?background:rgb\(255,0,0\)/);
```

Comment:
could this match the property if written on an other selector?
would it be possible to extract the code for the selector with regex and then check if it includes the right property?

Reply:
Not sure I know what you mean, can you give an example? Or if you have a code suggestion I'm open to it.

I should say the same type of regex was already used elsewhere.

Reply:
![image](https://user-images.githubusercontent.com/26656284/184440862-cedd64f2-a4e0-4b70-9731-cc50c667af60.png)

I don't think we really have to worry about it, but...

As regex, I would say something like `/\.red{.*?}/` and on the extracted text `.includes("background:rgb(255,0,0)")

Reply:
Just trying to think like a user here. Should probably allow for white space after the commas in (255,0,0\). While we are at it, probably between between the parens and the numbers as well.

Reply:
@ieahleen I see what you mean. That seems fairly unlikely to happen unless someone goes out of their way to do something that wasn't asked for.

There are 4 other challenge steps as well that are using that same regex (besides the 3 in this PR).

Reply:
It just came to mind, I don't think we have to worry about that too much